### ISSUE TITLE
Using styles located on a network drive

### ISSUE THREAD

**Issue by sverremb** [asciidoctor/brackets-asciidoc-preview#36]
I've got the following in my .adoc-files to enable the styles found in the template-url:

ifndef::template-url[]
:template-url: \\networkdrive\templates\Asciidoc
include::{template-url}\styles\core-styles.adoc[]
endif::[]

In the preview I get the following error message:
Unresolved directive in <stdin> - include::\\networkdrive\templates\Asciidoc\styles\core-styles.adoc[]

Does brackets asciidoc preview support what I'm trying to do here?

**Comment by nerk** [comment]
Try to use forward slashes, even on Windows.

ifndef::template-url[]
:template-url: \\networkdrive\templates\Asciidoc
include::{template-url}\styles\core-styles.adoc[]
endif::[]

**Comment by sverremb** [comment]
Thanks for the quick reply! 

Unfortunately, it didn't work.  I think I've tried every possible combination of back- and forward-slashes, even tried using the drive letter on the network drive.

**Comment by nerk** [comment]
You are right, I just tried it myself. It works with Asciidoctor, just not for the preview plugin. I think it might be related to prepending the user definable base directory (from the settings dialog). Could be a bug, I'll have to check.

**Comment by nerk** [comment]
This is a problem with asciidoctor.js. I created an issue, you can track it here: https://github.com/asciidoctor/asciidoctor.js/issues/273
